components/support/viaduct-reqwest/src/lib.rs

```diff
@@ -114,12 +114,3 @@ pub fn use_reqwest_backend() {
 pub extern "C" fn viaduct_use_reqwest_backend() {
     use_reqwest_backend();
 }
-
-/// A dummy symbol we include so that we can detect whether or not the reqwest
-/// backend got compiled in.
-#[no_mangle]
-pub extern "C" fn viaduct_detect_reqwest_backend() {
```

Comment:
this was from Tarik - @bendk, do you know whether this is actually safe to remove when considering desktop?

Reply:
I think so?  I don't see that when I search in searchfox.

Reply:
Thanks - I hope you mean *does* look safe to remove, so I'll mark this as ready

Reply:
I went spelunking, and it looks like Thom added this way back in #937, as a check to ensure that we never included the Reqwest backend in our (then-Android only) Megazords—and which Ed removed in #3191.